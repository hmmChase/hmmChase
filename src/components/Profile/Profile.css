.profile h2 {
  margin-top: 0;
  text-align: center;
}

.skills {
  display: flex;
  flex-flow: wrap;
  justify-content: center;
  width: 250px;
  margin: 0 auto;
}

.skill:nth-of-type(odd) {
  width: 150px;
}

.skill:nth-of-type(even) {
  width: 100px;
}

.skill {
  font-size: 1rem;
}

.platforms {
  display: flex;
  justify-content: center;
  font-size: 3rem;
}

.platform-icon {
  margin: 25px 25px 0 25px;
}

.platform-icon:visited {
  color: #000;
}
