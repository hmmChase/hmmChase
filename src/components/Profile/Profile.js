import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import './Profile.css';

const Profile = () => (
  <div>
    <h2>Web Developer</h2>

    <div className='skills'>
      <div className='skills-left'>
        <span className='skill'>JavaScript</span>
        <span className='skill'>jQuery</span>
        <span className='skill'>HTML</span>
        <span className='skill'>CSS/SCSS</span>
        <span className='skill'>Git</span>
        <span className='skill'>React</span>
        <span className='skill'>Router</span>
        <span className='skill'>Redux</span>

        <a className='platform-icon' href='https://github.com/hmmChase'>
          <FontAwesomeIcon icon={['fab', 'github']} />
        </a>
      </div>

      <hr className='split' />

      <div className='skills-right'>
        <span className='skill'>NextJS</span>
        <span className='skill'>Express</span>
        <span className='skill'>GraphQL</span>
        <span className='skill'>Postgres</span>
        <span className='skill'>Mocha</span>
        <span className='skill'>Chai</span>
        <span className='skill'>Jest</span>
        <span className='skill'>Enzyme</span>

        <a
          className='platform-icon'
          href='https://www.linkedin.com/in/chase-r/'
        >
          <FontAwesomeIcon icon={['fab', 'linkedin']} />
        </a>
      </div>
    </div>

    {/* <div className='platforms'>
          <a className='platform-icon' href='https://github.com/hmmChase'>
            <FontAwesomeIcon icon={['fab', 'github']} />
          </a>

          <a
            className='platform-icon'
            href='https://www.linkedin.com/in/chase-r/'
          >
            <FontAwesomeIcon icon={['fab', 'linkedin']} />
          </a>
        </div> */}
  </div>
);

export default Profile;
