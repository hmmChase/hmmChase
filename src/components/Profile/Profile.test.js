import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import Profile from './Profile';

describe('Profile', () => {
  it('matches snapshot', () => {
    const mockProps = {};

    const utils = render(<Profile {...mockProps} />);

    const container = utils.container;

    expect(container).toMatchSnapshot();
  });
});
