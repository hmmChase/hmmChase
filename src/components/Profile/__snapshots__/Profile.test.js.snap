// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile matches snapshot 1`] = `
<div>
  <h2>
    Web Developer
  </h2>
  <div
    class="skills"
  >
    <div
      class="skills-left"
    >
      <span
        class="skill"
      >
        JavaScript
      </span>
      <span
        class="skill"
      >
        jQuery
      </span>
      <span
        class="skill"
      >
        HTML
      </span>
      <span
        class="skill"
      >
        CSS/SCSS
      </span>
      <span
        class="skill"
      >
        Git
      </span>
      <span
        class="skill"
      >
        React
      </span>
      <span
        class="skill"
      >
        Router
      </span>
      <span
        class="skill"
      >
        Redux
      </span>
      <a
        class="platform-icon align-right"
        href="https://github.com/hmmChase"
      >
        <i
          class="fa-undefined"
        />
      </a>
    </div>
    <hr
      class="split"
    />
    <div
      class="skills-right"
    >
      <span
        class="skill"
      >
        NextJS
      </span>
      <span
        class="skill"
      >
        NodeJS
      </span>
      <span
        class="skill"
      >
        Express
      </span>
      <span
        class="skill"
      >
        GraphQL
      </span>
      <span
        class="skill"
      >
        Postgres
      </span>
      <span
        class="skill"
      >
        Jest
      </span>
      <span
        class="skill"
      >
        Mocha/Chai
      </span>
      <span
        class="skill"
      >
        Enzyme/RTL
      </span>
      <a
        class="platform-icon"
        href="https://www.linkedin.com/in/chase-r/"
      >
        <i
          class="fa-undefined"
        />
      </a>
    </div>
  </div>
</div>
`;
